presto-main/src/main/java/io/prestosql/server/security/UserMapping.java

```diff
@@ -84,33 +86,57 @@ public UserMappingRules(
         }
     }
 
+    enum Case
+    {
+        KEEP(Function.identity()),
+        LOWER(value -> value.toLowerCase(Locale.ENGLISH)),
+        UPPER(value -> value.toUpperCase(Locale.ENGLISH)),
+        /**/;
```

Comment:
No need for this here, unless we think we will be adding more later. The purpose of this construct is to have cleaner diffs when adding future items (as the previously last item won't be disturbed).